content/master/concepts/patch-and-transform.md

```diff
@@ -0,0 +1,1692 @@
+    fromFieldPath: spec.field1
+    toFieldPath: metadata.annotations["myAnnotation"]
+    transforms:
+      - type: match
+        match:
+          patterns:
+            - type: literal
+              literal: "UnknownString"
+              result: "foundField1"
+          fallbackTo: Value
+          fallbackValue: "StringNotFound"
+```
+
+To use the original input as the fallback value use 
+{{<hover label="defaultInput" line="12">}}fallbackTo: Input{{</hover>}}.
+
+Crossplane uses the original 
+{{<hover label="defaultInput" line="3">}}fromFieldPath{{</hover>}} input for the 
+{{<hover label="defaultInput" line="4">}}toFieldPath{{</hover>}} value.
+```yaml {label="defaultInput"}
+patches:
+  - type: FromCompositeFieldPath
+    fromFieldPath: spec.field1
+    toFieldPath: metadata.annotations["myAnnotation"]
+    transforms:
+      - type: match
+        match:
+          patterns:
+            - type: literal
+              literal: "UnknownString"
+              result: "foundField1"
+          fallbackTo: Input
+```
+
+### Math transforms
+
+Use the {{<hover label="math" line="6">}}math{{</hover>}} transform to multiply
+an input or apply a minimum or maximum value. 
+
+{{<hint "important">}}
+A {{<hover label="math" line="6">}}math{{</hover>}} transform only supports
+integer inputs. 
+{{< /hint >}}
+
+```yaml {label="math",copy-lines="1-7"}
+patches:
+  - type: FromCompositeFieldPath
+    fromFieldPath: spec.numberField
+    toFieldPath: metadata.annotations["mathAnnotation"]
+    transforms:
+      - type: math
+        math:
+          ...
+```
+
+<!-- vale Google.Headings = NO -->
+#### clampMin
+<!-- vale Google.Headings = YES -->
+
+The {{<hover label="clampMin" line="8">}}type: clampMin{{</hover>}} uses a defined 
+minimum value if an input is larger than the 
+{{<hover label="clampMin" line="8">}}type: clampMin{{</hover>}} value.
+
+For example, this 
+{{<hover label="clampMin" line="8">}}type: clampMin{{</hover>}} requires an
+input to be greater than 
+{{<hover label="clampMin" line="9">}}20{{</hover>}}.
+
+If an input is lower than 
+{{<hover label="clampMin" line="9">}}20{{</hover>}}, Crossplane uses the 
+{{<hover label="clampMin" line="9">}}clampMin{{</hover>}} value for the 
+{{<hover label="clampMin" line="4">}}toFieldPath{{</hover>}}.
+
+```yaml {label="clampMin"}
+patches:
+  - type: FromCompositeFieldPath
+    fromFieldPath: spec.numberField
+    toFieldPath: metadata.annotations["mathAnnotation"]
+    transforms:
+      - type: math
+        math:
+          type: clampMin
+          clampMin: 20
+```
+
+<!-- vale Google.Headings = NO -->
+#### clampMax
+<!-- vale Google.Headings = YES -->
+
+The {{<hover label="clampMax" line="8">}}type: clampMax{{</hover>}} uses a defined 
+minimum value if an input is larger than the 
+{{<hover label="clampMax" line="8">}}type: clampMax{{</hover>}} value.
+
+For example, this 
+{{<hover label="clampMax" line="8">}}type: clampMax{{</hover>}} requires an
+input to be greater than 
```

Comment:
i think this is supposed to be "less than" 🤔

Reply:
MATH IS HARD, OKAY!? (also fixed)